client/flutter/lib/pages/protect_yourself.dart

```diff
@@ -19,12 +19,12 @@ class ProtectYourself extends StatelessWidget {
           titleWidget: EmojiHeader("🚷"), message: "Avoid crowded places"),
       CarouselSlide(context,
           titleWidget: EmojiHeader("🏠"),
-          message: "Stay at home if you feel unwell - even with a slight fever and cough"),
-
+          message:
```

Comment:
is this formatted correctly?

Reply:
This is how `flutter format . --fix` does it. I think the real problem here is that throughout the code base **trailing commas** are *not* used. See the [Flutter style guide](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#use-a-trailing-comma-for-arguments-parameters-and-list-items-but-only-if-they-each-have-their-own-line). I could open a PR for that if you think that it is a problem.

So adding a trailing comma here would fix that formatting awkwardness.